src/main/java/frc/robot/commands/BangBangSubsystemCommand.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.DriveSub;
+
+public class BangBangSubsystemCommand extends Command{
+    private final DriveSub subsystem;
+    private final double setpoint;
+    private final double power = 0.5;
```

Comment:
Power of 0.5 is a lot for the subsystem. You need to test different values, but 0.01 might be good.

Reply:
fixed it